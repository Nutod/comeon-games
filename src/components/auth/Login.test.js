describe("Login", () => {
  expect(true).toBe(true)
})