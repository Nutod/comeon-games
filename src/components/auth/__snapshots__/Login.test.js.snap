// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> Snapshot Should match snapshot 1`] = `
<Route>
  <Component />
</Route>
`;
